Polyfills/Console/Source/Console.cpp

```diff
@@ -34,16 +38,136 @@ namespace
+                {
+                    matching = false;
+                    std::string_view prefix(firstArg.data() + start, j - start + 1);
+                    // This is a substitution argument, so if we have any remaining arguments to substitute, we should.
+                    // Try to use the next arg for subsitution
+                    if (currArgIndex < info.Length())
+                    {
+                        Napi::Value currArg = info[currArgIndex];
+                        // Check the type of the sub string
+                        if (EndsWith(prefix, 'f'))
+                        {
+                            double d = currArg.ToNumber().DoubleValue();
+                            if (std::isnan(d))
+                            {
+                                ss << "NaN";
+                            }
+                            else
+                            {
+                                // I had to copy the string view to a string here, because the data pointer of the string view
+                                // returns a view of the entire character sequence, which caused an issue on the following
+                                // step when formatting.
+                                std::string copiedSubString(prefix);
+                                std::string formatted = FormatValue(copiedSubString, d);
+                                ss << formatted;
+                            }
+                        }
+                        else if (EndsWith(prefix, 'd') || EndsWith(prefix, 'i'))
+                        {
+                            // For some reason, converting to int doesn't result in nans, so I check with double.
+                            double d = currArg.ToNumber().DoubleValue();
+                            if (std::isnan(d))
+                            {
+                                ss << "NaN";
+                            }
+                            else
+                            {
+                                int64_t n = currArg.ToNumber().Int64Value();
+                                // Some explanation as above to why copy to a string
+                                std::string copiedSubString(prefix);
+                                std::string formatted = FormatValue(copiedSubString, n);
+                                ss << formatted;
+                            }
+                        }
+                        else // 'o', 'O', 's'
+                        {
+                            ss << currArg.ToString().Utf8Value();
+                        }
+                        currArgIndex++;
+                    }
+                    else
+                    {
+                        ss << prefix;
+                    }
+                    start = j + 1;
+                }
+                else if (matching && (currChar == ' '))
```

Comment:
Similar to the previous comment, it is not guaranteed that there will be space and it doesn't make much sense to explicitly search for space.